var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -19,6 +19,7 @@ class PyProtobuf(PythonPackage):
 
     variant("cpp", default=False, description="Enable the cpp implementation")
 
+    version("4.21.5", sha256="eb1106e87e095628e96884a877a51cdb90087106ee693925ec0a300468a9be3a")
     version("3.20.1", sha256="adc31566d027f45efe3f44eeb5b1f329da43891634d61c75a5944e9be6dd42c9")
```

Comment:
```suggestion
    version("3.20.1", sha256="adc31566d027f45efe3f44eeb5b1f329da43891634d61c75a5944e9be6dd42c9", preferred=True)
```

Reply:
@adamjstewart Do you want to merge my suggestion to see if it goes through without forcing a new CI run and approval process?

Reply:
Let's see if the current solution passes CI. If it does, we can merge without pinning